### DISCUSSION TITLE
New Setting Items

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#734] [Ideas]
**Global custom explorer**
- for aka Total Commander, Directory opus, Files (UWP)
- https://github.com/Flow-Launcher/Flow.Launcher/discussions/729
- We have already this feature some plugin, but explorer plugin doesn't.
- This setting needs to be separated to public.
- in setting window, It is necessary to show an example of how to put it in the setting.
- It would be nice to be able to automatically find a route and show a list.

**Recommendation Hotkey**
- It would be nice if you could choose a list rather than a direct input.
- Of course, leave the existing one.
- List : Alt Space, Pause, Scroll Lock, Twice press Ctrl(Listary's Default), Win+Capslock.

**Showing lnk file Path**
- It named 'Hide the installed program path.' (better title?)
- It's Checkbox in file plugin. Default is 'uncheck'
- usually, *.lnk files doesn't need display path.
- If Check, *.lnk file showing path.
- UWP needs to be added in the case.

**Comment by onesounds** [comment]
**Open Window Position**
- In now. Active Monitor Center
- Need to add Last Position and selectable Monitor (Primary, Secondary), Top right, Top Left, ETC.

**Comment by jjw24** [comment]
>Recommendation Hotkey

list is good

>Showing lnk file Path

seems reasonable

>Open Window Position

something powertoys has, and yeah would be good addition
